samples/subsys/display/lvgl/sample.yaml

```diff
@@ -40,3 +40,36 @@ tests:
       - mimxrt595_evk_cm33
     integration_platforms:
       - mimxrt1170_evk_cm7
+  sample.subsys.display.lvgl.stm32:
+    # This test case is intended to insure that this sample builds & runs
+    # correctly for STM32 boards that have a supported built-in display.
+    filter: dt_chosen_enabled("zephyr,display")
+    platform_allow:
+      - stm32f429i_disc1
+      - stm32f746g_disco
+      - stm32f7508_dk
+      - stm32h7b3i_dk
+    harness: console
+    harness_config:
+      fixture: fixture_display
+    modules:
+      - lvgl
+    tags:
+      - samples
+      - display
+      - lvgl
+      - gui
+  sample.subsys.display.lvgl.st_b_lcd40_dsi1_mb1166:
+    platform_allow: stm32h747i_disco_m7
+    extra_args: SHIELD=st_b_lcd40_dsi1_mb1166
+    harness: console
+    harness_config:
+      fixture: fixture_display
+    modules:
+      - lvgl
+    tags:
+      - samples
+      - display
+      - shield
+      - lvgl
+      - gui
```

Comment:
Can you maybe harmonize the tags across the entries? Some of them feature a `gui` tag some don't.